### PR TITLE
Goto choose avatar

### PR DESCRIPTION
**Pull request by MarckK**

### CODE REVIEW

File: src/controllers/breathing.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ const breatheCtrl = () => {
   else {
     breathingMenu.setHideMenuTimer(7000);
     startNotificationSequence();
+    // localStorage.setItem('hasVisited', 'true');
```
Comment (gabrielperales) [inline src/controllers/breathing.js]: why this is being removed?, it should be changed by `saveState({hasVisited: true})` but not deleted/removeed

File: src/index.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ const fullscreenHandler = () => {
 );
 
 window.addEventListener('load', () => setTimeout(() => {
-  const breathingPageVisited = localStorage.getItem('hasVisited');
+  const breathingPageVisited = localStorage.getItem('state'); //changed from 'hasVisited'
```
Comment (gabrielperales) [inline src/index.js]: we have this module https://github.com/CYPIAPT-LNDSE/breathing-with-kitty/blob/master/src/globalState.js to manage the state. We shouldn't be using `localStorage` out of that module.

File: src/animations/index.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ export const welldoneToIntro = () => {
     .add(TweenMax.to('#welldone-stars', 0.3, { scale: 0 }))
     .add(TweenMax.to('#welldone-banner', 0.3, { y: -60, opacity: 0 }))
     .add(TweenMax.to('.welldone-user', 0.2, { opacity: 0 }))
-    .add(TweenMax.fromTo('.breathing', 1, { opacity: 0 }, { opacity: 1 }));
+    // .add(TweenMax.fromTo('.breathing', 1, { opacity: 0 }, { opacity: 1 }));
```
Comment (gabrielperales) [inline src/animations/index.js]: I just wanted to put that meme 💃

File: src/controllers/breathing.js (hunk 1 above)
Comment (MarckK) [reply src/controllers/breathing.js]: Hi, we aren't using 'hasVisited' anymore; we are just going to query 'state' in local storage to check if user has visited.

**Pull request merged**